docs/infrastructure/administration.md

```diff
@@ -312,14 +312,16 @@ CREATE USER 'myuser' IDENTIFIED BY 'PASSWORD >=30 CHARACTERS LONG' REQUIRE SSL;
 -- Tables where we want to allow only partial access.
 -- `password` is randomly generated by Django and never used/exposed due to SSO,
 -- so is not really sensitive, but it causes less confusion to still exclude it.
-GRANT SELECT (id, username, email) ON treeherder.auth_user to 'myuser';
+GRANT SELECT, SELECT (id, username, email) ON treeherder.auth_user to 'myuser';
```

Comment:
This is really weird! I'm not as familiar with MySql so I don't know why that double SELECT would make sense.

Reply:
I *think* this is a `GRANT` command with two, comma separated, values.  The first allows the user to perform SELECT queries (the ability to consume server resources, with nothing said about what tables), the second is allowing specific table/columns.   This command is not SQL.

Reply:
So... is it something specific to MySQL?

Reply:
Yes, it is specific to the MySQL permissions system.  Although the other database vendors have something **very** similar; with differences in what resources are available.  Maybe it has been standardized?

Reply:
Ok, well either way let's get it added to our docs.